src/module/mod.rs

```diff
@@ -132,24 +133,24 @@ impl Module {
         let mut ret = Module::default();
         ret.config = config.clone();
         let mut indices = IndicesToIds::default();
-        let mut validator = Validator::new();
-        validator.wasm_features(WasmFeatures {
-            reference_types: !config.only_stable_features,
-            multi_value: true,
-            bulk_memory: !config.only_stable_features,
-            simd: !config.only_stable_features,
-            threads: !config.only_stable_features,
-            multi_memory: !config.only_stable_features,
-            ..WasmFeatures::default()
-        });
+
+        // For now we have the same set of wasm features
+        // regardless of config.only_stable_features.
```

Comment:
Perhaps we can add a note that new unstable features may be added in future under `only_stable_features: false`:

```suggestion
        // For now we have the same set of wasm features
        // regardless of config.only_stable_features. New unstable features
        // may be enabled under `only_stable_features: false` in future.
```